cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/AbstractReadyResourceOperator.java

```diff
@@ -39,6 +41,10 @@ public AbstractReadyResourceOperator(Vertx vertx, C client, String resourceKind)
         super(vertx, client, resourceKind);
     }
 
+    public Future<Void> readiness(String namespace, String name, long pollIntervalMs, long timeoutMs) {
```

Comment:
It's worth pointing out that this could be moved up the inheritance hierarchy (up to `AbstractResourceOperator`, in fact), since it's not specific to resources with readiness: It's just a generalised poll-based wait method with a timeout. It doesn't have to be done in this PR.